Docs/Configuration.tex

```diff
@@ -6814,9 +6814,11 @@ \subsection{OpenLinuxBoot}\label{uefilinux}
   as distro boots successfully.
 \end{itemize}
 
-If using OpenLinuxBoot with Secure Boot, users may wish to use the \texttt{shim-to-cert.tool} included in OpenCore
-utilities, which can be used to extract the public key needed to boot a distro's kernels directly, as done when using OpenCore
-with OpenLinuxBoot, rather than via GRUB shim. For non-GRUB distros, the required public key must be found by user research.
+If using OpenLinuxBoot with Secure Boot, users may wish to install
+an own-signed, own-build Shim bootloader giving SBAT and MOK integration, as explained in
```

Comment:
`self-signed` and `self-built`?

Reply:
Well, I guess (more or less not explicitly) I must have avoided self-signed because the software isn't signed by itself, but by the owner...

Reply:
i.e. if 'self-signed' didn't already mean something else (imo) I would have used it

Reply:
I'll reword and avoid these.